mlflow/langchain/__init__.py

```diff
@@ -127,6 +128,28 @@ def get_default_conda_env():
     return _mlflow_conda_env(additional_pip_deps=get_default_pip_requirements())
 
 
+def _get_databricks_serverless_env_vars():
+    """
+    To initialize WorkspaceClient correctly in a subprocess in databricks, some
+    environment variables need to be set.
+
+    Note:
+        Databricks authentication related environment variables are set in the
+        _capture_imported_modules function.
```

Comment:
Can we also set `SPARK_LOCAL_REMOTE` in `_capture_imported_modules`?

Reply:
This is only needed when WorkspaceClient is initialized inside subprocess, adding it in `_capture_imported_modules` would add it for all cases